### DISCUSSION TITLE
iptables rules are not persistent after reboot

### DISCUSSION THREAD

**Discussion by ponte19** [Security-Onion-Solutions/securityonion#11746] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

24

### RAM

128 GB

### Storage for /

1TB

### Storage for /nsm

1TB

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No errors (i selected the wrong thing)

### Logs

No, there are no additional clues

### Detail

I am refering to this ticket: https://github.com/Security-Onion-Solutions/securityonion/discussions/11466

I want to edit firewall rules but on every restart, the rules disappear.
After adding certain rules to the chains in iptables I did
```
sudo su
iptables-save > /etc/sysconfig/iptables
```

but it still does not save the rules.

For me it is important to block this 169.254.169.254 address, too.
I need to keep firewall logs clean and this address messes up my logs completely!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by ponte19** [comment]
I found out, that reboots are not necessary. The rule deletes itself after some time?!

**Comment by dougburks** [comment, marked as answer]
Firewall rules are managed by Security Onion. If you want to edit firewall rules, please see:
https://docs.securityonion.net/en/2.4/firewall.html#host-firewall
